partner_aging/wizard/supplier.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import tools
+from odoo import api, fields, models, _
+
+
+class SupplierAgingDate(models.TransientModel):
+    _name = "supplier.aging.date"
+
+    age_date = fields.Datetime("Aging Date", required=True,
+                               default=lambda self: fields.Datetime.now())
+
+    @api.multi
+    def open_partner_aging(self):
+        ctx = self._context.copy()
+        ctx.update({'age_date': self.age_date})
+
+        supplier_aging = self.env['partner.aging.supplier.ad']
+        supplier_aging.execute_aging_query(age_date=self.age_date)
+
+        return {
+            'name': _('Supplier Aging'),
+            'view_type': 'form',
+            'view_mode': 'tree,form',
+            'res_model': 'partner.aging.supplier.ad',
+            'type': 'ir.actions.act_window',
+            'domain': [('total', '!=', 0)],
+            'context': ctx,
+        }
+
+
+class PartnerAgingSupplierAD(models.Model):
```

Comment:
I feel like this duplicates a significant amount of code with the customer side of things. I would recommend using an abstract model to unify this